@shared/api/types.ts

```diff
@@ -101,5 +102,3 @@ declare global {
 }
 
 export type CURRENCY = { code: string; issuer: string; image: string };
-
-export type CustomNetwork = NetworkDetails & { isSwitchSelected: boolean };
```

Comment:
before I had `background` adding a custom network and then switching to that network if `isSwitchSelected`. Now I just have the UI call 2 services, `addCustomNetwork` and `changeNetwork` if that's what the user wants